glide.lock

```diff
@@ -169,4 +184,63 @@ imports:
   - pkg/util/yaml
   - pkg/watch
   - pkg/watch/versioned
+- name: k8s.io/kubernetes
```

Comment:
this is why I would like this to be separate PR/issue so we can discuss this without blocking secrets